src/20_Components/amp-install-serviceworker.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  #### Introduction
+
+  The amp-install-serviceworker component is used to install [Service Workers](https://www.w3.org/TR/service-workers/), either on the current origin or a third-party origin.
+
+  Having an AMP document install a Service Worker enables the bootstraping of a [Progressive Web App](https://developers.google.com/web/progressive-web-apps) from an AMP doc.
+-->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <!-- #### Setup -->
+  <!--
+    Import the amp-install-serviceworker component
+  -->
+  <script async custom-element="amp-install-serviceworker" src="https://cdn.ampproject.org/v0/amp-install-serviceworker-0.1.js"></script>
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+
+  <!-- #### Basic Usage -->
+  <!--
+    The amp-install-serviceworker component requests that the runtime install a Service Worker.  This service worker could be used to cache the app shell of your Progressive Web App (PWA).
+  -->
+    <amp-install-serviceworker 
+        src="https://example.com/serviceworker.js" 
+        layout="nodisplay"></amp-install-serviceworker>
+  <!-- #### Third-Party Origin -->
+  <!--
+    If the AMP document is being served from a third-party AMP cache, such as the Google AMP Cache, you'll need to link to the URL of an HTML document to iframe in to install the service worker.
+    In this case, the HTML document could simply be an AMP HTML document itself using the amp-install-serviceworker component.
```

Comment:
Will the iframed doc be visible? Are there any recommendations what it should include? Can we simply link to this example?